db/model.js

```diff
@@ -6,4 +6,21 @@ module.exports.topics = require("./data/development-data/topics.js")
 
 module.exports.fetchArticleById = function(id) {
   return db.query("SELECT * FROM articles WHERE article_id = $1;", [id])
+}
+
+module.exports.fetchArticles = function() {
+  return db.query("SELECT * FROM articles ORDER BY created_at DESC;")
+}
+
+module.exports.fetchCommentsByArticle = function(id) {
+  const sql = `SELECT * FROM comments WHERE article_id = ${id};`
+  //console.log(sql)
+  return db.query(sql)
+}
+
+module.exports.articleComments = function() {
+  return db.query(
+    `SELECT articles.article_id, comments.comment_id 
+    FROM articles JOIN comments
+    ON articles.article_id = comments.article_id;`)
 }
```

Comment:
These can and should be combined into one query. You can and should also had a .then(...) to control what you send back with any data manipulation (which ideally shouldnt need to happen as the query should just give back what you want)